pkg/go.mod

```diff
@@ -0,0 +1,36 @@
+module github.com/k8ssandra
```

Comment:
I think I prefer to have a single `go.mod` for the whole repo.

Reply:
This will be easier to add after the tests has been merged.

Reply:
👍